api/hasura/actions/_handlers/createUsers.ts

```diff
@@ -153,6 +155,20 @@ async function handler(req: VercelRequest, res: VercelResponse) {
       `panic: insertedUsers is null; ${JSON.stringify(mutationResult, null, 2)}`
     );
 
+  let profileId: number;
+  if (sessionVariables.hasuraRole == 'user') {
+    profileId = sessionVariables.hasuraProfileId;
+  }
+
+  insertedUsers.map(async inviteduserId => {
```

Comment:
let's use `await Promise.all(insertedUsers.map...)` here so we don't have a delay time that grows linearly with the number of users... or better yet, is there a syntax for batch-inserting in a single hasura request?

Reply:
you should be able to batch insert like this.
```ts
insert_interaction_events: [
  { object: newEvents },
  { returning: {id: true  }}
]
```

Reply:
i made this use the multiple insert model throughout with a varargs pattern